packages/components/input/src/input.vue

```diff
@@ -224,9 +225,20 @@ const wrapperKls = computed(() => [
 
 const attrs = useAttrs({
   excludeKeys: computed<string[]>(() => {
-    return Object.keys(containerAttrs.value)
+    const _attrs = Object.keys(containerAttrs.value)
```

Comment:
@btea When I think about why this is happening, it seems to have something to do with the lack of maxlength and minlength properties.

It seems that further investigation into the solution is needed.

Reply:
Yes, if the user adds the `countGraphemes` attribute, the calculation of the maxlength of characters may be inconsistent with the default behavior, so it needs to be removed here.

There are a lot of special emoji, and their lengths are not uniform, so they can't match the maxlength boundary value very well. I think this seems to be troublesome. 🤔